extensions/toolbox/src/script/date/timestamp.ts

```diff
@@ -0,0 +1,26 @@
+import { Script } from "../type";
+
+export const toTimestamp: Script = {
+  info: {
+    title: "Date to Timestamp",
+    desc: "Converts dates to Unix timestamp",
```

Comment:
`"Convert dates to Unix timestamps"` would be better here.

Reply:
Thank you. I fixed it.